README.md

```
@@ -38,3 +38,6 @@ flowchart LR
 
 
 ````
+
+
+some modification
```

markdownlint-cli2 (0.22.1)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)